.ci.yaml

```diff
@@ -383,7 +383,8 @@ targets:
     recipe: engine/engine_lint
     properties:
       add_recipes_cq: "true"
-      cores: "12"
+      cores: "8"
```

Comment:
This dimension should be removed, but when I do the try falls back to 12 in the upstreamed config and fails with no resource https://chromium-swarm.appspot.com/task?id=619612e614a28f10&o=true&w=true.

The ones in prod are also 8 core so this should still run there.  I'll remove it when the config syncs after merging.

Reply:
Yeah, all arms are with 8 cores. Why not removing the cores from this PR?

Reply:
> Yeah, all arms are with 8 cores. Why not removing the cores from this PR?

When I remove it the build fails with resource unavailable: when I do the try falls back to 12 in the upstreamed config and fails with no resource https://chromium-swarm.appspot.com/task?id=619612e614a28f10&o=true&w=true.

I assumed it's not passing in "no cores", it's falling back to the synced config that says 12?  I was thinking I would merge this and then try to remove the cores part in a future PR.

Reply:
I'm going to merge this and then remove the cores dimension in the next PR.